cleanlab/multilabel_classification/dataset.py

```diff
@@ -0,0 +1,93 @@
+import pandas as pd
+import numpy as np
+from cleanlab.filter import _find_multilabel_issues_per_class
+from cleanlab.internal.multilabel_utils import get_onehot_num_classes
+from collections import defaultdict
+
+
+def common_multilabel_issues(
+    labels=list,
+    pred_probs=None,
+    *,
+    class_names=None,
+    confident_joint=None,
+) -> pd.DataFrame:
+    """Returns the Tags that are often mis-labeled.
+
+
+    This method provides two scores in the Pandas DataFrame that is returned:
+
+    * **num_examples**: The number of examples where the given/predicted label differ.
+    * **Issue_probability**: `(num_examples / total number of examples in the dataset`).
+
+    This method works by providing any one (and only one) of the following inputs:
+
+    1. ``labels`` and ``pred_probs``, or
+    2. ``confident_joint``
+
+    Only provide **exactly one of the above input options**, do not provide a combination.
+
+    Parameters
+    ----------
+    labels : List[List[int]]
+        Refer to documentation for this argument in filter._find_multilabel_issues_per_class() for further details.
+
+    pred_probs : np.ndarray, optional
+      Refer to documentation for this argument in filter._find_multilabel_issues_per_class() for further details.
+
+
+    class_names : Iterable[str]
+        A list or other iterable of the string class names. The list should be in the order that
+        matches the label indices. So if class 0 is 'dog' and class 1 is 'cat', then
+        ``class_names = ['dog', 'cat']``.
+
```

Comment:
you are missing docstring for the `confident_joint` argument. Make sure to clarify what this should look like from our other multi-label confident joint docstrings.